Update readme and genesis files (#593)
PolymathNetwork/Polymesh: README.md

cargo build --release
```

Run unit tests:

```bash
./scripts/test.sh
```

# Branches

> The `develop` branch is the working branch with the latest code changes.
> 
> The `alcyone` branch tracks code deployed to the Polymesh Alcyone Public Testnet.
> 
> The `tooling` branch tracks the next candidate release for the Polymesh Alcyone Public Testnet.
> 

Reviewer: ```suggestion
- The `develop` branch is the working branch with the latest code changes.
- The `alcyone` branch tracks code deployed to the Polymesh Alcyone Public Testnet.
- The `tooling` branch tracks the next candidate release for the Polymesh Alcyone Public Testnet.
```

## Build

Binary should be built in *debug mode*, using `cargo build` without `--release` parameter:  

```bash
cargo build
```

Test cases are built in *debug mode* by default.

## Using GDB

> Using `rust-gdb` you will get more pretty printed values of different types than go directly with `gdb`.

Reviewer: ```suggestion
Using `rust-gdb` you will get pretty printed values for more types than directly with `gdb`.
```

> The following example, start `gdb`, set a breakpoint and start it:

Reviewer: ```suggestion
The following example, starts `gdb`, sets a breakpoint, and starts our compiled `polymesh`:
```
